awscript.sh

```diff
@@ -2,9 +2,9 @@
 
 # cat ips | xargs -I% sh -c "host %"
 # -z True if it's a zero | -n True if it's not a zero!
-[ -z $1 ] && echo "#Usage: awscript.sh <domains/ips>" && exit 
+[ -z $1 ] && { echo "#Usage: awscript.sh <domains/ips>" >&2; exit 1; }
```

Comment:
Slight logic improvement, by grouping together relevant commands.

Reply:
This is a professional work and I like it.